network/datadog_checks/network/network.py

```diff
@@ -298,7 +304,8 @@ def _check_linux(self, instance):
                     # bug that print `tcp` even if it's `udp`
                     # The `-H` flag isn't available on old versions of `ss`.
                     cmd = "ss --numeric --tcp --all --ipv{} | cut -d ' ' -f 1 | sort | uniq -c".format(ip_version)
-                    output, _, _ = get_subprocess_output(["sh", "-c", cmd], self.log)
+                    output, _, _ = get_subprocess_output(["sh", "-c", cmd], self.log,
+                                                         env={"PROC_ROOT": net_proc_base_location})
```

Comment:
(nit) `{"PROC_ROOT": net_proc_base_location}` is used twice, could be saved in a variable to avoid duplication.

Reply:
Done!